com.borgbase.vorta.yaml

```diff
@@ -0,0 +1,88 @@
+app-id: com.borgbase.vorta
+
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+
+command: vorta
+
+finish-args:
+  # X11/Wayland
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+  # Sync
+  - --share=network
+  # Filesystem access
+  - --filesystem=home
+  - --filesystem=/run/media
+  - --filesystem=/media
+  # dconf access
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  # without it, no tray icon is displayed
+  - --own-name=org.kde.*
+  # without it mounting does not work
+  - --own-name=org.freedesktop.*
```

Comment:
Well, that's too much as a permission. Same for --own-name=org.kde.*. You don't really own none of the two buses.  You should use --talk-name & you might want to take a look at how apps with tray icons handle that.

Reply:
Yeah, I also found [this](https://github.com/flathub/flathub/wiki/App-Requirements#ownership) and yesterday I asked at the flatpak IRC channel for help, but unfortunately I got no responses. I think I looked at the manifest of pretty much all apps who use the system tray and I found two examples who also had to rely on "own-name": [qBittorrent](https://github.com/flathub/org.qbittorrent.qBittorrent/pull/25) and [Viber](https://github.com/flathub/com.viber.Viber/issues/4#issuecomment-382122314).

I tried some variations of "--talk-name" but none of them worked. I am currently out of options and any help would greatly be appreciated.

Reply:
When I mount without `--own-name=org.freedesktop.*`, I get:

```
Mounting filesystem
Initializing llfuse
Calling fuse_mount
Portal call failed: org.freedesktop.DBus.Error.ServiceUnknown
```

Reply:
Thank you for your suggestions, I hope the manifest is fine now :)

Reply:
Well.. sadly the KDE StatusNotifier spec is broken and you do need that KDE ownership kinda...

But usually the specific hardcoded StatusNotifier name works (although its extremely likely to conflict and there is nothing you can do about it without patching how StatusNotifier works)

Reply:
Is there a bug report to subscribe to reg. the brokenness of StatusNotifier?